python/dask_cudf/dask_cudf/accessors.py

```diff
@@ -1,6 +1,46 @@
 # Copyright (c) 2021, NVIDIA CORPORATION.
 
 
+# from cudf.python.dask_cudf.dask_cudf.core import Index
```

Comment:
```suggestion
```

Reply:
Thanks, Rick!!  Not sure how this slipped through..  My mistake!